tests/test_models/test_forward.py

```diff
@@ -45,6 +45,69 @@ def _get_detector_cfg(fname):
     return model, train_cfg, test_cfg
 
 
+def test_sparse_rcnn_forward():
```

Comment:
Use `@pytest.mark.skipif(not torch.cuda.is_available(), reason="requires gpu")` as decorator.